.buildkite/pipeline.yml

```
@@ -84,3 +84,30 @@ steps:
     depends_on:
       - step: "tests"
         allow_failure: false
+
+  - label: ":gcloud: Release test"
+    key: "release-test"
+    command: ".buildkite/scripts/release_test.sh"
+    agents:
+      provider: "gcp"
+    depends_on:
+      - step: "tests"
+        allow_failure: false
+
+  - label: ":jenkins: Release - Package Registry Distribution"
+    key: "release-package-registry"
+    command: echo "Triggering the Jenkins Job 'release-package-registry-distribution'..."
+    agents:
+      provider: "gcp"
+    depends_on:
+      - step: "release-test"
+        allow_failure: false
+
+  - label: ":jenkins: Downstream - Package"
+    key: "downstream-package"
+    command: echo "Triggering the Jenkins Job 'fleet-server/fleet-server-package-mbp/env.JOB_BASE_NAME'..."
+    agents:
+      provider: "gcp"
+    depends_on:
+      - step: "release-package-registry"
+        allow_failure: false
```

As these two steps are just placeholders for the time being, we could just use k8s agents instead of creating new VMs for those two echo commands. It could be removed the agent key definition:

```suggestion
  - label: ":jenkins: Release - Package Registry Distribution"
    key: "release-package-registry"
    command: echo "Triggering the Jenkins Job 'release-package-registry-distribution'..."
    depends_on:
      - step: "release-test"
        allow_failure: false

  - label: ":jenkins: Downstream - Package"
    key: "downstream-package"
    command: echo "Triggering the Jenkins Job 'fleet-server/fleet-server-package-mbp/env.JOB_BASE_NAME'..."
    depends_on:
      - step: "release-package-registry"
        allow_failure: false
```

---

fixed